dogpile/cache/backends/memcached.py

```diff
@@ -112,6 +112,11 @@ class GenericMemcachedBackend(CacheBackend):
     to the :meth:`set` method."""
 
     def __init__(self, arguments):
+        # No need to override serializer, as all the memcached libraries will do it.
+        # Still, we support customizing the serializer/deserializer to use better default pickle protocol
```

Comment:
but we want to use our own pickler here don't we ?

Reply:
Well, this can potentially break backwards compatibility (I have to double check that), but probably in a way that nobody will ever notice.

It seems to me that all the memcached libraries use flags that represent the type of value stored. For example, [this](https://github.com/linsomniac/python-memcached/blob/bad41222379102e3f18f6f2f7be3ee608de6fbff/memcache.py#L966) is how python-memcached does it. Other memcached implementations (even the C ones) seem to do the same. If we suddently start passing always bytes to memcached client, then it will be suddently start using no flag, to signify that bytes are stored.
This can potentially be a problem for users that expect Client._FLAG_TEXT to be set for some values.
But given the behaviour of memcached to return None on GET if there are errors, it may not be such a big deal after all.

But this is the only reason why I left the default to noop, so to leave the memcached library to determine what to do.

Reply:
OK so then.... we don't support any means of changing the serializer right now?  They can default to None for serializer/deserializer but we still need to run our "(de)serializer" inside the get/set methods in the backend right?